### PR TITLE
Add GrowByData Services Pvt. Ltd.

### PR DESCRIPTION
**Pull request by Saphall**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/growbydata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "GrowByData Services Pvt. Ltd.",
+  "establishedAt": 1646072100,
```
Comment (thenishantsapkota) [inline data/growbydata.json:3]: Everything looks good, but can I get the source of this company establishment date?

**Review by TheLearneer — COMMENTED** [review]
@Saphall can you please update the remote branch and then add the fields `phone` and `email` 🙏

File: data/growbydata.json (hunk 1 above)
Comment (Saphall) [reply data/growbydata.json:3]: @thenishantsapkota Got to know though this article : https://omgnepal.com/growbydata-data-revolution-in-nepal/

File: data/fusemachines.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "establishedAt": 1356977700,
   "location": "Devkota Sadak, Baneshwor, Kathmandu",
   "website": "https://fusemachines.com/",
-  "email": null,
+  "email": "info@fusemachines.com",
```
Comment (Saphall) [inline data/fusemachines.json:6]: Updated the email through their contact info : https://fusemachines.com/contact/

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contributions @thenishantsapkota partying_face
Join our [Discord](http://discord.gg/7jwZaa8WDr) to claim your contributors role (if you haven't already) and stay connected with us.
Cheers!

**Comment by thenishantsapkota** [comment]
Thank you for your contributions @Saphall  :partying_face: 
Join our [Discord](http://discord.gg/7jwZaa8WDr) to claim your contributors role (if you haven't already) and stay connected with us.
Cheers!

**Pull request merged**